codestyle/styles.md

```diff
@@ -0,0 +1,672 @@
+# CSS/LESS code style
```

Comment:
I think, there should be written something about modular structure. I.e. one block per file, for instance.